### DISCUSSION TITLE
Railway 部署

### DISCUSSION THREAD

**Discussion by bottomash** [y-young/nazurin#52] [General]
heroku即将不再支持免费部署，[railway](railway.app) 是一个类似heroku的平台。

railway每个月支持5美刀的免费额度，fork项目直接部署并在添加环境变量后即可使用。

railway部署应该是可行的一种部署方案。

**Comment by y-young** [comment]
理论上是可行的，缺点是未验证信用卡的账户每个月有500小时的运行时间限制，所以无法支持一整个月的持续运行，此外每个月开始时限额需要重新部署之后才能重置。
